api/chronos/serializers.py

```diff
@@ -26,6 +26,8 @@ class Meta:
     name = ser.CharField(read_only=True)
     title = ser.CharField(read_only=True)
 
+    links = LinksField({'self': 'get_absolute_url'})
```

Comment:
Do you mind adding the links field to the ChronosSubmissionSerializer as well? It was missing in both places.